installer/main.sh

```diff
@@ -22,19 +22,34 @@ DNS_DOMAIN_NAME=$2
 GCE_ZONE=$3
 MINIO_ACCESS_KEY="${MINIO_ACCESS_KEY:=my_minio_key}"
 MINIO_SECRET_KEY="${MINIO_SECRET_KEY:=my_minio_secret}"
-MINIO_URL=minio.$DNS_DOMAIN_NAME
+MINIO_SIGNATURE="${MINIO_SIGNATURE:=s3v4}"
+MINIO_REGION="${MINIO_REGION:=us-east-1}"
+
+if [[ -z "${MINIO_URL}" ]]; then
+  MINIO_URL="http://minio.$DNS_DOMAIN_NAME"
+  INSTALL_MINIO=true
+else
+  INSTALL_MINIO=false
+fi
+
+if [[ $MINIO_URL == *"https"* ]];
```

Comment:
i'd change that to:
if [[ $MINIO_URL =~ ^https ]];

(since we want to check whether https is at the beggining)

Reply:
additionally this could be shorter:
[[ $MINIO_URL =~ ^https ]] && export USE_HTTPS=1 || export USE_HTTPS=0

(but then we should change rest of the similar if-then-else expr)